app/models/classification.rb

```diff
@@ -46,6 +46,27 @@ class Classification < ApplicationRecord
 
   FIXTURE_FILE = FIXTURE_DIR.join("classifications.yml")
 
+  def self.parent_classifications
```

Comment:
The existing method `categories` does a similar thing but is scoped to the current region. Perhaps you can modify that method to get all the categories across all regions?

Reply:
the method `categories` lists (with these changes) categories with `read_only=true &&  mapped categories` from other regions. So it would not help in UI to display them in tag assignments.